packages/core/ai_content_pipeline/ai_content_pipeline/pipeline/step_executors/video_steps.py

```
@@ -0,0 +1,354 @@
+            )
+
+
+class UpscaleVideoExecutor(BaseStepExecutor):
+    """Executor for video upscaling steps."""
+
+    def execute(
+        self,
+        step,
+        input_data: Any,
+        chain_config: Dict[str, Any],
+        step_context: Optional[Dict[str, Any]] = None,
```

Ruff (0.14.10)

Unused method argument: `step_context`

(ARG002)